app/scripts/auth.js

```diff
@@ -35,14 +40,28 @@ export const isParticipant = connectedReduxRedirect({
   wrapperDisplayName: 'UserIsParticipant',
 })
 
+export const isParticipantOrFreeFestival = connectedReduxRedirect({
+  redirectAction,
+  redirectPath,
+  authenticatedSelector: state => {
```

Comment:
To not use the same check multiple times you could write:

```
state => {
  const isFestivalFree = state.meta.config.festivalTicketPrice === 0
  return ...
}
```

Reply:
fixed here: https://github.com/adzialocha/hoffnung3000/commit/c98d4e5caa3a794b630d5447aca5bd514c00980f